css/questions.css

```diff
@@ -335,6 +301,9 @@
 		line-height: 18px;
 	}
 }
+.form__label_checkbox p{
+	text-decoration: underline;
```

Comment:
for `<p>` ?
I've always thought that it needs for `<a>`.

UPD 
I checked the markup, it must be a link of course.
And it must lead hear
![image](https://user-images.githubusercontent.com/37267977/219847197-46f18e9b-57b6-465f-9557-ee431f403fdd.png)

(check all the places)

Reply:
Done.

Reply:
Please, every time when I write comments, find all such places on the site.

Reply:
I didn't find any such places. Done.